src/main/scala/wdl4s/WdlExceptions.scala

```diff
@@ -0,0 +1,15 @@
+package wdl4s
+
+import scalaz.NonEmptyList
+
+trait ThrowableWithErrors extends Throwable {
+  def message: String
+  def errors: NonEmptyList[String]
+
+  override def getMessage = {
+    s"""$message\n${errors.list.mkString("\n")}"""
+  }
+}
+
+class UnsatisfiedInputsException(message: String) extends RuntimeException(message)
```

Comment:
Not really - it was already there so I assumed it made sense to whoever wrote it.